### PR TITLE
add container status

### PR DESCRIPTION
**Pull request by xlgao-zju**
A part of #8 

Signed-off-by: Xianglin Gao <xlgao@zju.edu.cn>

---
This change is [Reviewable](https://reviewable.kubernetes.io/reviews/kubernetes/frakti/21)

### CODE REVIEW

File: pkg/hyper/helper.go [hunk 1]
```diff
@@ -211,19 +212,52 @@ func inMap(in, dest map[string]string) bool {
 	return true
 }
 
+// Len is a method for Sort to compute the length of s.
 func (s sandboxByCreated) Len() int {
 	return len(s)
 }
 
+// Len is a method for Sort to compute while one is less between two items of s.
```
Comment (feiskyer) [inline pkg/hyper/helper.go]: Len -> Less

File: pkg/hyper/helper.go [hunk 2]
```diff
@@ -211,19 +212,52 @@ func inMap(in, dest map[string]string) bool {
...
 func (s sandboxByCreated) Less(i, j int) bool {
 	return *s[i].CreatedAt > *s[j].CreatedAt
 }
 
+// Len is a method for Sort to swap the items in s.
```
Comment (feiskyer) [inline pkg/hyper/helper.go]: Len -> Swap

File: pkg/hyper/helper.go [hunk 3]
```diff
@@ -211,19 +212,52 @@ func inMap(in, dest map[string]string) bool {
...
 func (s sandboxByCreated) Swap(i, j int) {
 	s[i], s[j] = s[j], s[i]
 }
 
-//Sort sort podSandboxList by creation time (newest first).
+// sortByCreatedAt sort podSandboxList by creation time (newest first).
```
Comment (feiskyer) [inline pkg/hyper/helper.go]: sort -> sorts

File: pkg/hyper/hyper.go [hunk 4]
```diff
@@ -238,7 +238,89 @@ func (h *Runtime) ListContainers(filter *kubeapi.ContainerFilter) ([]*kubeapi.Co
...
+		Labels:      kubeletLabels,
+		Annotations: annotations,
+		CreatedAt:   &status.CreatedAt,
+	}
+
+	mounts := make([]*kubeapi.Mount, len(status.Container.VolumeMounts))
+	for idx, mnt := range status.Container.VolumeMounts {
+		mounts[idx] = &kubeapi.Mount{
+			Name:          &mnt.Name,
+			ContainerPath: &mnt.MountPath,
```
Comment (feiskyer) [inline pkg/hyper/hyper.go]: Missing HostPath

**Comment by xlgao-zju** [comment]
@feiskyer updated

**Comment by feiskyer** [comment]
LGTM. Thanks.

**Pull request merged**